gm4_orbis/data/gm4_orbis/functions/structure/chest_dungeon.mcfunction

```diff
@@ -6,3 +6,22 @@ execute store result score chest_height gm4_count run data get entity @s Pos[1]
 execute if score chest_height gm4_count matches ..29 run data merge block ~ ~ ~ {LootTable:"gm4_orbis:chests/dungeon_20"}
 execute if score chest_height gm4_count matches 30..39 run data merge block ~ ~ ~ {LootTable:"gm4_orbis:chests/dungeon_30"}
 execute if score chest_height gm4_count matches 40.. run data merge block ~ ~ ~ {LootTable:"gm4_orbis:chests/dungeon_40"}
+
+# set the loot table seed
+data modify storage gm4_orbis:temp Pos set from entity @s Pos
+execute store result score $x gm4_orbis_config run data get storage gm4_orbis:temp Pos[0] 64
+execute store result score $y gm4_orbis_config run data get storage gm4_orbis:temp Pos[1] 64
+execute store result score $z gm4_orbis_config run data get storage gm4_orbis:temp Pos[2] 64
+data remove storage gm4_orbis:temp Pos
+
+scoreboard players operation $loot_table_seed gm4_orbis_config = #world_seed gm4_orbis_config
+scoreboard players operation $loot_table_seed gm4_orbis_config *= $x gm4_orbis_config
+scoreboard players operation $loot_table_seed gm4_orbis_config *= $y gm4_orbis_config
+scoreboard players operation $loot_table_seed gm4_orbis_config *= $z gm4_orbis_config
+
+execute store result block ~ ~ ~ LootTableSeed long 256 run scoreboard players get $loot_table_seed gm4_orbis_config
+
+scoreboard players reset $x gm4_orbis_config
+scoreboard players reset $y gm4_orbis_config
+scoreboard players reset $z gm4_orbis_config
+scoreboard players reset $loot_table_seed gm4_orbis_config
```

Comment:
I'm not a fan of all this resetting storage and scoreboards when it serves no purpose. It's less efficient for no gain.

Reply:
That's what I assumed, but I thought @Bloo-dev liked keeping debug stuff clean